magicctapipe/scripts/lst1_magic/README.md

```diff
@@ -0,0 +1,234 @@
+    LST-3: 0
+    LST-4: 0
+    MAGIC-I: 2
+    MAGIC-II: 3
+
+directories:
+    workspace_dir : "/fefs/aswg/workspace/yourname/yourprojectname/"  # Output directory where all the data products will be saved.
+    # MC paths below are ignored if you set NSB_matching = true.
+    MC_gammas     : "/fefs/aswg/data/mc/DL0/LSTProd2/TestDataset/sim_telarray"  # set to "" if you don't want to process these Monte Carlo simulations.
+    MC_electrons  : "" 
+    MC_helium     : "" 
+    MC_protons    : "/fefs/aswg/data/mc/DL0/LSTProd2/TrainingDataset/Protons/dec_2276/sim_telarray" 
+    MC_gammadiff  : "/fefs/aswg/data/mc/DL0/LSTProd2/TrainingDataset/GammaDiffuse/dec_2276/sim_telarray/"
+    
+data_selection:
+    source_name_database: "CrabNebula"  # MUST BE THE SAME AS IN THE DATABASE; Set to null to process all sources in the given time range.
+    source_name_output: 'Crabtest'  # Name tag of your target. Used only if source_name_database != null.
+    target_RA_deg : 83.629  # RA in degrees; Please set it to null if source_name_database=null.
+    target_Dec_deg: 22.015  # Dec in degrees; Please set it to null if source_name_database=null.
+    time_range : True  # Search for all runs in a LST time range (e.g., 2020_01_01 -> 2022_01_01).
+    min : "2021_01_17"
+    max : "2022_12_03"   
+    date_list : ['2020_12_15','2021_03_11']  # LST list of days to be processed (only if time_range=False), format: YYYY_MM_DD.
+    skip_LST_runs: [3216,3217]  # LST runs to ignore.
+    skip_MAGIC_runs: [5094658]  # MAGIC runs to ignore.
+    
+general:
+    SimTel_version: "v1.4"  
+    LST_version   : "v0.10"
+    LST_tailcut   : "tailcut84"
+    focal_length  : "effective"
+    proton_train_fraction  : 0.8  # 0.8 means that 80% of the DL1 protons will be used for training the Random Forest.
+    nsb           : [0.5, 1.0, 1.5, 2.0, 2.5, 3.0]
+    env_name      : auto_MCP_PR  # name of the conda environment to be used to process data.
+    cluster       : "SLURM"  # cluster management system on which data are processed. At the moment we have only SLURM available, in the future maybe also condor (PIC, CNAF).
+    NSB_matching  : true  # Set to false to process also the MCs. Set to true if adequate MC productions (DLx) are already available on the IT Container.
+    NSB_MC        : 0.5  # extra noise in dim pixels used to process MCs; e.g., you could put here the average NSB value of the processed LST runs. Ignored if NSB_matching=true.
+    
+    
+```
+
+Now that the configuration file is ready, let's create a list with all the MAGIC+LST1 runs for the time window (or list of nights) defined on the config_general.yaml file:
+
+> $ list_from_h5 -c config_general.yaml
+
+The output in the terminal should look like this:
+```
+Cleaning pre-existing *_LST_runs.txt and *_MAGIC_runs.txt files
+Finding LST runs...
+Finding MAGIC runs...
+```
+And it will save the files TARGET_LST_runs.txt, TARGET_MAGIC_runs.txt, and list_sources.dat in your working directory.
```

Comment:
what if no runs are found? are the files created anyway but with nothing in them? or will the `list_from_h5` script say something about that?

Reply:
It should not create the files. I will check this and add a warning if needed

Reply:
I added a return in the script in case no run can be found. I wlll add also a note in the readme